src/modules/auto-scroll-to-id.js

```diff
@@ -27,8 +27,10 @@
 		}
 		
 		if (!!offsetSelector) {
-			var offsetItem = $(offsetSelector).eq(0);
-			offset = offsetItem.height() * -1;
+			var offsetItem = $(offsetSelector).each(function () {
```

Comment:
The `offsetItem` does not have any usage now that we are doing it in a loop...

```suggestion
			$(offsetSelector).each(function () {
```